src/script/backup/CrossPlatformBackup/CPB.helper.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import Dexie from 'dexie';
+
+import {ClientEvent} from 'src/script/event/Client';
+import {getLogger} from 'Util/Logger';
+
+import {CPBackupImporter, BackupPeekResult} from './CPB.library';
+
+import {BackupService} from '../BackupService';
+import {IncompatibleBackupError} from '../Error';
+
+export const CPBLogger = getLogger('wire:backup:CPB');
+
+export const isCPBackup = async (data: ArrayBuffer | Blob): Promise<boolean> => {
+  if (data instanceof ArrayBuffer) {
+    const backupImporter = new CPBackupImporter();
+    const backupData = new Uint8Array(data);
+    const result = await backupImporter.peekFileData(backupData);
+    if (result instanceof BackupPeekResult.Failure) {
+      return false;
+    }
+    if (result instanceof BackupPeekResult.Success) {
+      CPBLogger.log(`Backup version: ${result.version}`);
+      return true;
+    }
+    return false;
+  }
+  return false;
```

Comment:
I'll prefer early return, so doe will be more readable ;)

```suggestion
  if (!(data instanceof ArrayBuffer)) {
    return false;
  }
    const backupImporter = new CPBackupImporter();
    const backupData = new Uint8Array(data);
    const result = await backupImporter.peekFileData(backupData);
    if (result instanceof BackupPeekResult.Failure) {
      return false;
    }
    if (result instanceof BackupPeekResult.Success) {
      CPBLogger.log(`Backup version: ${result.version}`);
      return true;
    }
    return false;
```

Reply:
addressed here c2f6d7f098786de969220185f631a7450a1e2e7d